### PR TITLE
Adding milestone applier for k/website 1.28

### PR DESCRIPTION
**Pull request by mickeyboxell**
Adding milestone applier for k/website 1.28.

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -247,6 +247,7 @@ blockades:
 - repos:
   - kubernetes/kubernetes
   blockregexps:
+
```
Comment (cblecker) [inline config/prow/plugins.yaml:250]: Why is this new line here?

**Review by cblecker — CHANGES_REQUESTED** [review]
/hold

File: config/prow/plugins.yaml (hunk 1 above)
Comment (reylejano) [reply config/prow/plugins.yaml:250]: Once the new line is removed, this PR should be ready
I verified that the 1.28 milestone is present for k/website and the `dev-1.28` branch is created in k/website

Comment (mickeyboxell) [reply config/prow/plugins.yaml]: Removed the new line. It looks like it was fat fingered.

**Comment by reylejano** [comment]
/lgtm
@cblecker , I think we can remove the hold on this with the removal of the newline

**Comment by mickeyboxell** [comment]
@cblecker Would you mind taking another look and removing the hold?

**Review by cblecker — APPROVED** [review]
/hold cancel

**Pull request merged**